app/src/test/java/com/concordia/mcga/factories/FloorFactoryTest.java

```diff
@@ -0,0 +1,146 @@
+        int expectedY = 2;
+        Mockito.when(expectedCursor.moveToNext()).thenReturn(false);
+        Mockito.when(expectedCursor.getInt(Mockito.anyInt())).thenReturn(expectedX, expectedY);
+        Mockito.when(mockDB.rawQuery(Mockito.anyString(), Mockito.any(String[].class))).thenReturn(expectedCursor);
+
+        // Execute
+        instance.insertWalkablePaths(testBuilding, 4, mockDB, mockMap);
+
+        // Verify
+        Mockito.verify(mockMap, Mockito.times(0)).makeWalkable(new IndoorMapTile(expectedX, expectedY));
+        Mockito.verify(expectedCursor).close();
+    }
+
+    @Test
+    public void testCreateWalkablePaths_pathsExist() throws Exception {
+        // Test Data
+        IndoorMapFactory instance = new IndoorMapFactory();
+        String name = "TEST_NAME";
+        String shortName = "SHORT_NAME";
+        Building testBuilding = new Building(new LatLng(0,0),name, shortName, new MarkerOptions());
+
+        // Mocking
+        SQLiteDatabase mockDB = Mockito.mock(SQLiteDatabase.class);
+        TiledMap mockMap = Mockito.mock(TiledMap.class);
+        Cursor expectedCursor = Mockito.mock(Cursor.class);
+
+        int expectedX = 1;
+        int expectedY = 2;
+        Mockito.when(expectedCursor.moveToNext()).thenReturn(true,false);
+        Mockito.when(expectedCursor.getInt(Mockito.anyInt())).thenReturn(expectedX, expectedY);
+        Mockito.when(mockDB.rawQuery(Mockito.anyString(), Mockito.any(String[].class))).thenReturn(expectedCursor);
+
+        // Execute
+        instance.insertWalkablePaths(testBuilding, 4, mockDB, mockMap);
+
+        // Verify
+        Mockito.verify(mockMap).makeWalkable(new IndoorMapTile(expectedX, expectedY));
+        Mockito.verify(expectedCursor).close();
+    }
+
+    @Test
+    public void testGetTiledMap_mapExists() throws Exception {
+        // Test Data
+        IndoorMapFactory instance = new IndoorMapFactory();
+        String name = "TEST_NAME";
+        String shortName = "SHORT_NAME";
+        Building testBuilding = new Building(new LatLng(0,0),name, shortName, new MarkerOptions());
+
+        // Mocking
+        SQLiteDatabase mockDB = Mockito.mock(SQLiteDatabase.class);
+        TiledMap mockMap = Mockito.mock(TiledMap.class);
+        Cursor expectedCursor = Mockito.mock(Cursor.class);
+
+        int expectedX = 1;
+        int expectedY = 2;
+        Mockito.when(expectedCursor.moveToNext()).thenReturn(true);
+        Mockito.when(expectedCursor.getInt(Mockito.anyInt())).thenReturn(expectedX, expectedY);
+        Mockito.when(mockDB.rawQuery(Mockito.anyString(), Mockito.any(String[].class))).thenReturn(expectedCursor);
+
+        // Execute
+        TiledMap expectedMap = instance.createTiledMap(testBuilding, 4, mockDB);
+
+        // Verify
+        Mockito.verify(expectedCursor, Mockito.times(2)).getInt(Mockito.anyInt());
+        Mockito.verify(expectedCursor).close();
+    }
+
+    @Test
+    public void testGetTiledMap_mapDoesNotExist() throws Exception {
```

Comment:
The signature of JUnit tests really does not matter IMO. For all these, you are correct, but what truly matters about these tests is the test itself, not its signature.